StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetLPMConfirmFlowTests.swift

```diff
@@ -985,7 +985,7 @@ extension IntentConfirmParams: Equatable {
         // Sanity check to make sure when we add new properties, we check them here
         let mirror = Mirror(reflecting: lhs)
         let propertyCount = mirror.children.count
-        XCTAssertEqual(propertyCount, 7)
+        XCTAssertEqual(propertyCount, 8)
```

Comment:
If possible, I think it'd be valuable to add a test in here like testCard_OnlyCardInfo_WithDefaults or testCard_AllFields_WithDefaults where we set the default checkbox and confirm.